src/python/pants/engine/internals/graph_test.py

```diff
@@ -1150,8 +1150,8 @@ def sources_rule_runner() -> RuleRunner:
 
 def test_sources_normal_hydration(sources_rule_runner: RuleRunner) -> None:
     addr = Address("src/fortran", target_name="lib")
-    sources_rule_runner.create_files(
-        "src/fortran", files=["f1.f95", "f2.f95", "f1.f03", "ignored.f03"]
+    sources_rule_runner.write_files(
+        {f"src/fortran/{f}": f for f in ["f1.f95", "f2.f95", "f1.f03", "ignored.f03"]}
```

Comment:
Rather than writing the content as `f`, simpler is to write `""`

Reply:
Yea, thought I would stay as close to the current behaviour as possible. My docker tests for instance breaks with "" in a few cases..

Reply:
It's kind of weird to write the file name as the content. That's one of the reasons I didn't like `create_files()`. I recommend only writing `f` when it's absolutely necessary - and even then, I wonder if you could write something like `"foo"` instead

Reply:
Will do. 
In the docker case I referred to, I used "FROM base", as dummy value, as that was somewhat sensible in the context..